astrology.lic

```diff
@@ -168,7 +169,7 @@ class Astrology
     cast_spell(@rtr_data, settings)
     perc_time = Time.now - 61
     Flags.add('rtr-expire', get_data('spells').spell_data['Read the Ripples']['expire']) if rtr_active?
-    DRCMM.get_telescope(@tele_storage) if @have_telescope
+    get_telescope(@tele_storage) if @have_telescope
```

Comment:
💡 I like not having the prefix so the code is concise, but I've recently decided that, overall, I prefer to have it. It makes it easier for new contributors to figure out where helpers come from.

Not required for you to change this, just getting my thoughts out there.

Reply:
My plan is to make DRCMM's methods more robust and add them to astrology.lic. 

```
if @have_telescope
      case center_telescope_on(target)
      when /open it/
```

Reply:
I will do whatever you want me to do in this situation, just let me know. :)

Reply:
```
  def roll_bones(is_tied, container)
    if get_bones
      DRC.bput('roll my bones', 'roundtime')
      store_bones
    end
  end
```

Reply:
I love the plan.

As for the prefix, if you come back to this file for some reason please add it, no need to bother with another commit just for that change.

Reply:
I'm just waiting for all of these things to get merged, then I will update the whole.

Reply:
I reverted the changes as I would just be doing that in the next PR. No worries!